packages/core/EventManager.ts

```diff
@@ -279,6 +279,27 @@ export default class EventManager {
       });
     }
 
+    // Now we can delete the old booking and its references.
+    const bookingReferenceDeletes = prisma.bookingReference.deleteMany({
+      where: {
+        bookingId: booking.id,
+      },
+    });
+    const attendeeDeletes = prisma.attendee.deleteMany({
+      where: {
+        bookingId: booking.id,
+      },
+    });
+
+    const bookingDeletes = prisma.booking.delete({
```

Comment:
Only this delete is needed, attendee & bookingReference are in Cascade and will be auto-deleted 👍

Reply:
I realize that isn't the right thing to do as well. Just pushed the change